### DISCUSSION TITLE
CPU and memory issues encountered during deployment

### DISCUSSION THREAD

**Discussion by sayfeeling** [milvus-io/milvus#3816] [Q&A and General discussion]
Mine is an I9 cpu with 8G memory on a single node. Why is it still prompted that the CPU and memory resources are insufficient, and there is no other POD running. Can anyone help?

我的是I9的cpu，单节点8G内存，为什么仍然提示CPU和内存资源不足，并没有其他POD在运行，谁能帮助一下吗？

Warning  FailedScheduling  <unknown>            default-scheduler  0/3 nodes are available: 3 Insufficient cpu, 3 Insufficient memory.
  Warning  FailedScheduling  <unknown>            default-scheduler  0/3 nodes are available: 3 Insufficient cpu, 3 Insufficient memory.
  Warning  FailedScheduling  <unknown>            default-scheduler  0/3 nodes are available: 3 Insufficient cpu, 3 Insufficient memory.
  Warning  FailedScheduling  <unknown>            default-scheduler  0/3 nodes are available: 3 Insufficient cpu, 3 Insufficient memory.
  Warning  FailedScheduling  <unknown>            default-scheduler  0/3 nodes are available: 2 Insufficient cpu, 3 Insufficient memory.

**Comment by shiyu22** [comment]
Hi,
Which version of Milvus are you using? And where did this warning prompt come from? This doesn't look like a warning from Milvus.
